src/main/java/core/basesyntax/files/ReaderFile.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderFile implements Reader {
```

Comment:
```suggestion
public class ReaderImpl implements Reader {
```
И у Богдана в лекциях был нейминг подобно этому. Когда имплементируешь интерфейс то добавляешь к имени просто Impl. Возможно это не принципиально

Reply:
Как я понял Reader это абстракция и я из нее реализую класс ReaderFile именно File, просто тут же можно реализовать например ReaderDB  для чрения из БД и уже его передавать ReaderServise

Reply:
fixed